cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/IndexedTable.java

```diff
@@ -100,7 +100,7 @@
   /**
    * Column key used to store the existence of a row in the secondary index.
    */
-  private static final byte[] IDX_COL = {'r'};
+  public static final byte[] IDX_COL = {'r'};
```

Comment:
Since this will only be done one time (3.0 -> 3.1) and we will delete this code after 3.1, I almost think it is better to copy it.